firebase-firestore/src/main/java/com/google/firebase/firestore/local/SQLiteLruReferenceDelegate.java

```diff
@@ -148,20 +156,29 @@ private void removeSentinel(DocumentKey key) {
   @Override
   public int removeOrphanedDocuments(long upperBound) {
     int[] count = new int[1];
```

Comment:
Not related to this change, but it'd be nicer if we have a brief implementation comment here to explain why `int[]`, something like:

`// Using int[1] instead of int such that it is update-able from lambdas`.

Reply:
I'm gonna leave this as is for now. I think this is a common enough (anti-)pattern that we don't need to point it out explicitly. Thanks for the suggestion though!